app/ports/migrations/0007_alter_field_clt_version.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-08-06 11:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ports', '0006_add_clt_version_and_variants'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='clt_version',
+            field=models.CharField(default='', max_length=100),
```

Comment:
Do we need two migrations in the same PR? (Maybe we do because of the deployed version.)

Reply:
Yes, the deployed demo version needs that we put these migration files.

Reply:
We now have three. But there is a way to avoid this, we can replace them with a single file and then run a fake migration.

Reply:
I leave that up to you. I'm fine with any solution, so feel free to merge it yourself in the proper way.